Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -6802,6 +6804,14 @@ void WebPageProxy::viewIsBecomingVisible()
     protectedPageClient->viewIsBecomingVisible();
 }
 
+void WebPageProxy::viewIsBecomingInvisible()
+{
+    WEBPAGEPROXY_RELEASE_LOG(ViewState, "viewIsBecomingInvisible:");
```

Comment:
This might be redundant with existing logging?

Reply:
It is redundant with activity status update but we have a log for viewIsBecomingVisible so I figured adding a logging for when it becomes invisible as the opposite effect would be more consistent.